### PR TITLE
Update bug_report.yml

### PR DESCRIPTION
**Pull request by archzi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -56,8 +56,8 @@ This is an issue of the Clash core *per se*, not to the derivatives of Clash, li
       render: yaml
       label: "Clash config"
       description: "
-在下方附上 Clash core 脱敏后配置文件的内容
-Paste the Clash core configuration below.
+在下方附上 Clash core 配置文件，请确保配置文件中没有敏感信息（比如：服务器地址，密码，端口等）
+Paste the Clash core configuration file below, please make sure that there is no sensitive information in the configuration file (eg:server address/url, password, port, etc.)
```
Comment (Dreamacro) [inline .github/ISSUE_TEMPLATE/bug_report.yml:60]: `e.g.` and you forget the period.

Comment (archzi) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: fixed it, thank you!

Comment (Dreamacro) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: could you please add a space after `e.g.,` ?

Comment (archzi) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: @Dreamacro fixed it

**Pull request merged**